torch/csrc/autograd/FunctionsManual.cpp

```diff
@@ -2164,6 +2164,30 @@ Tensor split_backward(
   return split_with_sizes_backward(grads, split_sizes, dim, sym_sizes, options);
 }
 
+Tensor _nested_narrow_backward(
+    const Tensor& grad,
+    const Tensor& self,
+    int64_t dim,
+    const c10::SymInt& start,
+    const c10::SymInt& length) {
+  Tensor grad_input = at::zeros_like(self);
+  Tensor narrowed_grad = grad_input.narrow_symint(dim, start, length);
+  Tensor grad_values = at::_nested_get_values(grad);
+  Tensor narrowed_grad_values = at::_nested_get_values(narrowed_grad);
+  TORCH_INTERNAL_ASSERT(
+      grad_values.dim() == narrowed_grad_values.dim(),
+      "Bug encountered in _nested_narrow_backward(); please open an issue");
```

Comment:
nit: technically TORCH_INTERNAL_ASSERT already tells us to open an issue I think lol

Reply:
ah okay, will remove then